media-libs/grok/grok-9.7.6.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit cmake
+
+DESCRIPTION="World's Leading Open Source JPEG 2000 Codec"
+HOMEPAGE="https://github.com/GrokImageCompression/grok"
+SRC_URI="https://github.com/GrokImageCompression/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
LICENSE is wrong.

Reply:
whats the right one, please?

Reply:
See https://github.com/GrokImageCompression/grok/blob/master/LICENSE